### PR TITLE
Maintain acronym and update acronym expansion.

### PR DESCRIPTION
**Pull request by wmat**
Keep MMWP as is but update it's expansion to Machine-Mode When-no-PMP-match Policy.  This fixes Issue https://github.com/riscv/riscv-isa-manual/issues/1280

### CODE REVIEW

File: src/smepmp.adoc
Comment (kersten1) [inline src/smepmp.adoc:129]:
```suggestion
. With the current spec M-mode can access any memory region unless restricted by a PMP rule with the ``pmpcfg.L`` bit set. There are cases where this approach is overly permissive, and although it’s possible to restrict M-mode by adding PMP rules during the boot process, this can also be seen as a waste of PMP rules. Having the option to block anything by default, and use PMP as an allowlist for M-mode is considered a safer approach. This functionality may be used during the boot process or upon *PMP reset*, using initial register settings. +
```

**Review by kersten1 — APPROVED** [review]

**Comment by aswaterman** [comment]
Let’s wait for explicit approval from one of the spec authors before merging.

**Comment by gfavor** [comment]
To @mick@ics.forth.gr and @tech-tee@lists.riscv.org, please review this ePMP spec issue and (hopefully) approve this acronym update to address an RVI friendly language policy violation.

**Comment by wmat** [comment]
> To @[mick@ics.forth.gr](mailto:mick@ics.forth.gr) and @[tech-tee@lists.riscv.org](mailto:tech-tee@lists.riscv.org), please review this ePMP spec issue and (hopefully) approve this acronym update to address an RVI friendly language policy violation.

Can I get a quick review of this acronym update, @[mick@ics.forth.gr](mailto:mick@ics.forth.gr) and @[tech-tee@lists.riscv.org](mailto:tech-tee@lists.riscv.org)?  I'd like to be able to merge it before the whole spec goes out for internal review to reduce the chace of this violation being flagged during the review process.

**Comment by jjscheel** [comment]
The tech-tee group is no longer meeting.  It's been archived.  So, will need @mickflemm to comment directly.

**Comment by jjscheel** [comment]
As a gentle reminder, the "Friendly Terminology" policy [link](https://docs.google.com/document/d/1lmEMU_Gi1KRQB2PtbjUSgbtlx5frUK40AuMO1OICSEY/) which uses the "Inclusive Naming" community [link](https://inclusivenaming.org/) includes the word "whitelist" as a [Tier 1: Replace Immediately](https://inclusivenaming.org/word-lists/tier-1/) word. 

So, I hope our request to @mickflemm is to help find a replacement, right?

**Comment by kersten1** [comment]
Also ties in with this PR - https://github.com/riscv/riscv-isa-manual/pull/1381

**Comment by kersten1** [comment]
Its a PR so suggestion is right there. The issue is linked in Bill's first comment.

**Review by rsahita — APPROVED** [review]

**Comment by mickflemm** [comment]
Sorry I missed this one, I'm also ok with allowlist / denylist. Thanks for updating this.

**Pull request merged**
